packages/lesswrong/components/hooks/useDeviceKind.ts

```diff
@@ -0,0 +1,21 @@
+import { useMemo } from 'react';
+import { isClient } from '@/lib/executionEnvironment';
+import type { DeviceKind } from '@/components/ultraFeed/ultraFeedSettingsTypes';
+
+const SM_BREAKPOINT_PX = 600; // Keep in sync with theme breakpoints
+
+export const useDeviceKind = (): DeviceKind => {
```

Comment:
We have an `isMobile` which I think is more reliable than just checking the breakpoint